python/ray/data/tests/test_execution_optimizer.py

```diff
@@ -138,6 +138,7 @@ def test_split_blocks_operator(ray_start_regular_shared):
     planner = Planner()
     op = get_parquet_read_logical_op(parallelism=10)
     logical_plan = LogicalPlan(op)
+    logical_plan._set_context(DataContext.get_current())
```

Comment:
note: for many tests in this file, i have to manually call `set_context()` because we are not going through the plan optimizer plan, which automatically sets the context.